autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/daos/ClusterJdbcDao.java

```diff
@@ -28,6 +28,9 @@
 
 import br.com.autonomiccs.autonomic.plugin.common.enums.ClusterAdministrationStatus;
 
+/**
+ * This class contains methods that provide results based on queries at the 'cluster' table.

```

Comment:
I think that something similar to this description is better: 
"This class deals with operations regarding the ' cluster'  table."